pkg/gui/controllers/helpers/refs_helper.go

```diff
@@ -305,8 +289,36 @@ func (self *RefsHelper) CreateCheckoutMenu(ref string, branches []*models.Branch
 			return nil
 		},
 		Tooltip: self.c.Tr.CheckoutCommitTooltip,
+		Key:     'd',
 	})
 
+	if len(branches) > 0 {
+		menuItems = append(menuItems, lo.Map(branches, func(branch *models.Branch, index int) *types.MenuItem {
+			return &types.MenuItem{
+				LabelColumns: []string{self.c.Tr.Actions.CheckoutBranch, style.FgBlue.Sprint(branch.Name)},
+				OnPress: func() error {
+					self.c.LogAction(self.c.Tr.Actions.CheckoutBranch)
+					if err := self.c.Git().Branch.Checkout(branch.Name, git_commands.CheckoutOptions{}); err != nil {
+						return err
+					}
+					return self.c.WithWaitingStatus(self.c.Tr.LoadingCommits, func(gocui.Task) error {
+						return self.c.Refresh(
+							types.RefreshOptions{Mode: types.SYNC, Scope: []types.RefreshableView{types.COMMITS}},
+						)
+					})
+				},
+				Tooltip: self.c.Tr.CheckoutBranchTooltip,
+				Key:     rune(index + 1 + '0'), // Convert 1-based index to key
+			}
+		})...)
+	} else {
+		menuItems = append(menuItems, &types.MenuItem{
+			LabelColumns:   []string{style.FgDefault.SetStrikethrough().Sprint(self.c.Tr.Actions.CheckoutBranch)},
```

Comment:
Setting the style here is unnecessary, the DisabledReason takes care of this. So it's enough to say
```
Label: self.c.Tr.Actions.CheckoutBranch
```

Reply:
Oh; that must be a more general problem with your terminal emulator then, or maybe with its configuration.

Reply:
Ok, I will try it with other emulators, although I don't think, that my Wezterm is the problem, because the decoration is rendered correctly in other contexts.

Reply:
Inside lazygit, you mean? That would be very strange. An example to try is pressing `r` on the checked-out branch; this should show a menu with the first two entries disabled. Are these struck through for you?

Reply:
I found out, that the strikethrough is invisible in tmux for me - outside of tmux it is rendered correctly. 
But this not generally the case, inside of Neovim, strikethrough is rendered correctly also in tmux. Even if I use LazyGit from Neovim inside Tmux the style is correctly rendered.